src/nft/components/details/detailsV2/DataPageDescription.tsx

```diff
@@ -1,13 +1,18 @@
 import { Trans } from '@lingui/macro'
+import styled from 'styled-components/macro'
 
 import { Tab, TabbedComponent } from './TabbedComponent'
 
+const DescriptionContentContainer = styled.div`
```

Comment:
would this be a `TabbedContentContainer` given it is used in both description and details?

Reply:
The entire component is referred to as `Description` in the spec, figma, and jira